src/quill-editor.component.ts

```diff
@@ -74,14 +75,14 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
   @Input() minLength: number;
   @Input() required: boolean;
   @Input() formats: string[];
-  @Input() bounds: HTMLElement | string;
+  @Input() bounds: HTMLElement |  string;
```

Comment:
Sorry, this change is from VScode (my linter in autosave). There is not relevant to the commit.